### PR TITLE
update drone-hugo plugin to it's new repo - drone-plugins/drone-hugo

### PR DESCRIPTION
**Pull request by sofixa**
As discussed in https://github.com/drone-plugins/drone-hugo/issues/8

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
   generate:
-    image: golang:1.6
+    image: golang:1.11
     commands:
       - go run scripts/gen-language-content.go
 
   build:
-    image: webhippie/hugo:latest
-    commands:
-      - git clone https://github.com/drone/drone-hugo-theme.git themes/drone-hugo-theme
```
Comment (tboerger) [inline .drone.yml]: You dropped the fetch of the theme.

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "themes/drone-hugo-theme"]
```
Comment (tboerger) [inline .gitmodules]: Please don't add it as a submodule, we want to keep always the latest theme version.

File: content/drone-plugins/drone-gcr/index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ image: plugins/gcr
 ---
 
 {{% alert info %}}
-The GCR plugin is an extension around the Docker plugin. Please see the documentation for the [Docker plugin]({{< relref "drone-plugins/drone-docker/index.md" >}}) for other configuration options.
+The GCR plugin is an extension around the Docker plugin. Please see the documentation for the [Docker plugin]({{ relLangURL "/drone-plugins/drone-docker/index.md" }}) for other configuration options.
```
Comment (tboerger) [inline content/drone-plugins/drone-gcr/index.md]: Why relLangURL?

Comment (tboerger) [reply content/drone-plugins/drone-gcr/index.md]: Ah yeah, you are right... I forgot that the plugin index have been prepared to localization.

File: .gitmodules (hunk 2 above)
Comment (tboerger) [reply .gitmodules]: I would personally prefer a simple git clone before the Hugo plugin step.

**Comment by sofixa** [comment]
@tboerger is that OK for you? I basically:

* rollbacked the Hugo version to the one in webhippie/hugo:latest (0.39) because i had a hard time getting the right configuration with the current way i18n is done; this should probably be looked at some day
* i went back to using a standard git clone for the theme
* using the drone-hugo plugin
* updated all references to it with a proper link

Adrian

**Comment by sofixa** [comment]
That was my original intention as well, kinda forgot that updating your fork updates the PR based on it as well :D 

I'll roll that back as well sometime today, and after the drone-hugo stuff is merged resubmit another one for the .drone.yml changes, and then another one foe the drone-firebase plugin.

Adrian

**Comment by sofixa** [comment]
@tboerger it's done

File: content/drone-plugins/drone-hugo/index.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 ---
-date: 2018-03-02T00:00:00+00:00
+date: 2018-15-02T00:00:00+00:00
```
Comment (tboerger) [inline content/drone-plugins/drone-hugo/index.md]: Invalid date? yyyy-mm-dd...

Comment (sofixa) [inline content/drone-plugins/drone-hugo/index.md]: I don't know where it came from, so i fixed it back as it was

**Pull request closed without merging**